modules/internal/OwnedObject.chpl

```diff
@@ -289,23 +263,17 @@ module OwnedObject {
      */
     pragma "leaves this nil"
     pragma "nil from this"
+    @deprecated(notes="owned.release method is deprecated - please use :proc:`owned.release` type function instead")
```

Comment:
Just noting that this one uses a different wording than the others (due to 'method', e.g.).  The lack of articles here ("the owned.release method" / "the owned.release type function") feels a bit clunky as well.